scene/main/window.cpp

```diff
@@ -895,6 +878,22 @@ void Window::_make_transient() {
 	}
 }
 
+void Window::_set_transient_exclusive_child(bool p_clear_invalid) {
+	if (exclusive && visible && is_inside_tree()) {
+		if (!is_in_edited_scene_root()) {
+			// Transient parent has another exclusive child.
+			if (transient_parent->exclusive_child && transient_parent->exclusive_child != this) {
+				ERR_PRINT(vformat("Making child transient exclusive, but parent has another exclusive child. This node: %s, parent: %s, other node: %s", this->get_description(), transient_parent->get_description(), transient_parent->exclusive_child->get_description()));
```

Comment:
How about this? Bunch of repeating "window" but I think this would really help demystify what this means and whether it matters for end users.
```suggestion
				ERR_PRINT(vformat("Attempting to make child window transient exclusive, but the parent window already has another exclusive child. This window: %s, parent window: %s, current exclude child window: %s", this->get_description(), transient_parent->get_description(), transient_parent->exclusive_child->get_description()));
```

Reply:
Also, is "transient exclusive" really a property of the child window? The code reads like it's a "transient parent" and an "exclusive child", so this would be more trying to make a window an exclusive child of a transient parent which already has an exclusive child. The question is then what's a "transient parent" :P

Reply:
Maybe it should be `Attempting to make transient child window exclusive, but the parent window already has another exclusive child. This window: %s, parent window: %s, current exclude child window: %s`, since `transient` is mostly a property of the child (it can't outlast the parent), if it's worth mentioning `transient` at all, since for non-transient window there's no clear parent-child relationships.

Reply:
I guess "transient" kind of refers to their relationship then, rather than any individual node. So it probably doesn't add much information here, as you say. Let's drop it all together.